test/parallel/test-http2-server-session-destroy.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const h2 = require('http2');
+
+const server = h2.createServer();
+server.listen(0, common.mustCall(() => {
+  h2.connect(`http://localhost:${server.address().port}`, common.mustCall((session) => {
```

Comment:
nit: Using `common.localhostIPv4` instead `"localhost"` could be better.

Reply:
👌